<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgEditPingChart</class>
 <widget class="QDialog" name="DlgEditPingChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar destinos del ping</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0" colspan="3">
    <widget class="QLineEdit" name="host8LineEdit"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLineEdit" name="host4LineEdit"/>
   </item>
   <item row="10" column="3">
    <widget class="QToolButton" name="host10Button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QToolButton" name="host4Button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QLineEdit" name="host9LineEdit"/>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="host2Button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Host destino</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QLineEdit" name="host7LineEdit"/>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="host3Button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLineEdit" name="host5LineEdit"/>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QLineEdit" name="host10LineEdit"/>
   </item>
   <item row="7" column="3">
    <widget class="QToolButton" name="host7Button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QToolButton" name="host8Button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QPushButton" name="acceptButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="removeChartButton">
     <property name="text">
      <string>Borrar gr√°fico</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLineEdit" name="host6LineEdit"/>
   </item>
   <item row="5" column="3">
    <widget class="QToolButton" name="host5Button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="host1Button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="host1LineEdit"/>
   </item>
   <item row="9" column="3">
    <widget class="QToolButton" name="host9Button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QToolButton" name="host6Button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLineEdit" name="host2LineEdit"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="host3LineEdit"/>
   </item>
   <item row="11" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>host1LineEdit</tabstop>
  <tabstop>host1Button</tabstop>
  <tabstop>host2LineEdit</tabstop>
  <tabstop>host2Button</tabstop>
  <tabstop>host3LineEdit</tabstop>
  <tabstop>host3Button</tabstop>
  <tabstop>host4LineEdit</tabstop>
  <tabstop>host4Button</tabstop>
  <tabstop>host5LineEdit</tabstop>
  <tabstop>host5Button</tabstop>
  <tabstop>host6LineEdit</tabstop>
  <tabstop>host6Button</tabstop>
  <tabstop>host7LineEdit</tabstop>
  <tabstop>host7Button</tabstop>
  <tabstop>host8LineEdit</tabstop>
  <tabstop>host8Button</tabstop>
  <tabstop>host9LineEdit</tabstop>
  <tabstop>host9Button</tabstop>
  <tabstop>host10LineEdit</tabstop>
  <tabstop>host10Button</tabstop>
  <tabstop>acceptButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
